source/includes/_comparison.md

```diff
@@ -1,26 +1,29 @@
 ## Comparison
-Retrieves both `Readings` and the `interval`-th `Predictions` in the last `horizon` span specified.
+Retrieves both `Readings` and `Predictions` for an user specified period(`horizon`). Users are also able to select the set of `Predictions` based on the `interval` specified.
```

Comment:
'a user' instead of 'an user' (English grammar is ridiculous, it's based on whether it's a vowel *sound* rather than actually being whether it's a vowel following or not, so since user's pronunciation is "You-ser" it gets an 'a')

Reply:
^ never knew this :)